contracts/src/Nodes.sol

```diff
@@ -133,11 +133,14 @@ contract Nodes is ERC721, Ownable {
     Get all nodes regardless of their health status
      */
     function allNodes() public view returns (NodeWithId[] memory) {
-        uint32 totalNodeCount = _nodeIdCounter;
+        uint32 totalNodeCount = _nodeIdCounter - _startingNodeId;
```

Comment:
Could we also increment the Node ID by 100 with every new node? Leaves some flexibility in the future for any design changes where an organization runs multiple nodes/sub-nodes (or rotates their keys, etc). If this contract is being re-written anyway I suppose it doesn't matter if we do it now

Reply:
Think I might do this as a separate PR. This thing is too big as it is